changelog.d/16017.misc

```diff
@@ -0,0 +1 @@
+Move support for application service query parameter authorization behind a configuration option.
```

Comment:
This should probably be a `removal`?

Reply:
I wasn't sure about that because technically it's not being removed - just shoved behind a config flag. I'll change it now though!

Reply:
It also covers deprecations. I feel like hiding something in miscellaneous that an admin might need to handle is a bad idea.